src/in-view.js

```diff
@@ -17,40 +17,46 @@ const inView = () => {
     * each registry.
     */
     const interval = 100;
-    const triggers = ['scroll', 'resize', 'load'];
+    const triggers = ['scroll', 'resize'];
 
     /**
     * Maintain a hashmap of all registries, a history
     * of selectors to enumerate, and an options object.
     */
     let selectors = { history: [] };
-    let options   = { offset: {}, threshold: 0, test: inViewport };
+    let options   = { offset: {}, threshold: 0, test: inViewport, async: true };
 
     /**
     * Check each registry from selector history,
     * throttled to interval.
     */
     const check = throttle(() => {
         selectors.history.forEach(selector => {
-            selectors[selector].check();
+            const registry = selectors[selector];
+            if (options.async) {
+                let elements = [].slice.call(document.querySelectorAll(selector));
+                if (elements.length !== registry.elements.length) {
+                    registry.elements = elements;
+                }
```

Comment:
What does this check do? Is it meant to save time? To me, it looks like it's here to save an operation if the number of elements turns out to be the same number of elements we already have, but what if we have the same number of elements but there are all new?